civis/resources/_resources.py

```diff
@@ -433,12 +432,12 @@ def parse_api_spec(api_spec, api_version, resources):
     paths = api_spec['paths']
     classes = {}
     for path, ops in paths.items():
-        class_name, methods = parse_path(path, ops, api_version, resources)
-        class_name_lower = class_name.lower()
-        if methods and classes.get(class_name_lower) is None:
-            classes[class_name_lower] = type(str(class_name), (Endpoint,), {})
+        base_path, methods = parse_path(path, ops, api_version, resources)
+        class_name = to_camelcase(base_path)
+        if methods and classes.get(base_path) is None:
+            classes[base_path] = type(str(class_name), (Endpoint,), {})
```

Comment:
Should this use `base_path.lower()` instead of `base_path`? Or are we guaranteed that the API endpoint name will always be lower case?

Reply:
I think `base_path.lower()` is better.